content/en/blog/2020/wasm-announce/index.md

```diff
@@ -49,8 +49,7 @@ incentivized developers to upstream extensions into Envoy that were used by only
 percentage of deployments, just to piggyback on its release mechanisms.
 
 Over time some of the most performance-sensitive features of Istio have been upstreamed
-into Envoy - [policy checks on traffic](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/rbac_filter),
-and [telemetry reporting](/docs/reference/config/telemetry/metrics/), for example.
```

Comment:
Do you want to replace it with https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/jwt_authn_filter, so that we have more than one example?

Reply:
sgtm